src/utils/hooks/api/vaults/use-get-vault-transactions.tsx

```diff
@@ -16,6 +15,19 @@ import { getCurrencyEqualityCondition } from '@/utils/helpers/currencies';
 
 import { useGetCurrencies } from '../use-get-currencies';
 
+type TransactionStatus = 'pending' | 'cancelled' | 'completed' | 'confirmed' | 'received' | 'retried';
```

Comment:
These types were in the unused TransactionHistory component. That component's been removed now so the types have been moved here.(From memory, TransactionHistory was built but there was an issue with the data so we reverted to the previous version of the vault transaction tables).

Reply:
This is out of scope of this PR, but maybe it would make sense to merge this type with the `ExtrinsicStatus` type we use in the transactions hook, what do you think?  @danielsimao